ServerCommunication/ResolveObject.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Messerli.ServerCommunication
+{
+    public class ResolveObject

```

Comment:
How about `ResolvedObject` with a `d`?

Reply:
ObjectToResolve it will be tomorrow

Reply:
Addressed in https://github.com/messerli-informatik-ag/server-communication/pull/3/commits/f5c3d91f0d115f09bfbd58baad29b0cb02e16f27